user_service_interface.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+
+from user import User
+
+
+class IUserManagementService(ABC):
```

Comment:
For this, since there is some code involved, you should create a folder `user_management_service`, in one file there have this interface, and every other file will be the implementation (one of them is the dict one).